code/modules/surgery/organs/external/_external_organ.dm

```diff
@@ -254,6 +254,14 @@
 	restyle_flags = EXTERNAL_RESTYLE_FLESH
 
 	bodypart_overlay = /datum/bodypart_overlay/mutant/snout
+	///Do we have a specific sprite we should pull?
```

Comment:
the way you word this makes it sound like a bool. change this to mention that this is the icon state (and what icon it's from)